index.html

```diff
@@ -208,5 +208,52 @@ <h3 class="about-me-card__title card-title">Skills</h3>
           </div>
         </section>
       </section>
+
+      <!-- contact -->
+      <section id="contact" class="contact-section">
+        <div class="section-container">
+          <h1 class="section-title">
+            I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+          </h1>
+          <form method="POST" action="https://formspree.io/f/xoqzgjwv">
+            <input type="text" placeholder="Full name" name="name" maxlength="30" required>
+            <input type="email" placeholder="Email address" name="email" required>
+            <textarea cols="30" rows="10" name="message" placeholder="Hey I’d like to connect to..." required></textarea>
```

Comment:
- 🙏Please add the validation for the `maxlength` attribute to the `<textarea>` element with the value `maxlength="500"`. This is a project requirement aimed at limiting 😵 the number of characters a user can enter within the `textarea` to a maximum of 500 characters. By enforcing this constraint, we ensure that the entered text does not exceed the specified limit.